modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_user_commands.adoc

```diff
@@ -171,6 +172,88 @@ NOTE: Once users are disabled, their connected browsers will be disconnected. Us
 | The user name.
 |====
 
+== List User Home Directories
+
+List all available root directories for user homes that are currently in use. For details see xref:configuration/user/user_management.adoc#moving-the-user-home[Moving the User Home] documentation.
+
+This command is complementary when using `user:move`.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:home:list-dirs [options]
+----
+
+=== Options
+
+[width="100%",cols="20%,70%"]
+|===
+| `--output=[OUTPUT]`
+| Output format (plain, json or json_pretty, default is plain) [default: "plain"].
+|===
+
+=== Examples
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:home:list-dirs
+  - /var/www/owncloud/data
+----
+
+== List all Users For a Given Home Directory
+
+List all users that have their home in a given path. For details see xref:configuration/user/user_management.adoc#moving-the-user-home[Moving the User Home] documentation.
+
+This command is complementary when using `user:move`.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:home:list-users [options] [--] <path>
+----
+
+=== Arguments
+
+[width="100%",cols="20%,70%"]
+|===
+| `path`
+| Path where the user home must be located
+|===
+
+=== Options
+
+[width="100%",cols="20%,70%"]
+|===
+| `--output=[OUTPUT]`
+| Output format (plain, json or json_pretty, default is plain) [default: "plain"].
+|===
+
+=== Examples
+
+Note for the example below, some user accounts are originated from LDAP.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:home:list-users /var/www/owncloud/data
+  - admin
+  - aca4c3ec-691d-103b-8380-55a4da3d3a76
+  - 9918b614-6a2e-103b-89a7-f5edf5d332f5
+  - c298ae18-6a2e-103b-89a8-f5edf5d332f5
+  - dbcca7b4-7306-103b-813a-19652cf0a9d2
+----
+
+You can combine `user:home:list-dirs` and `user:home:list-users` which lists all users from all available home directories. The example shows, that user `lisa` has been moved to a different home directory with `user:move`.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:home:list-dirs | \
+  sed -e 's/  - //' | tr '\n' '\0' | xargs -0 -t -n 1 \
+  occ user:home:list-users
```

Comment:
See core PR https://github.com/owncloud/core/pull/39583
I have added a `--all` option to `occ user:home:list-users` which does this.

That should be easier than the "magic" here. Hopefully it will be approved, and then we can modify this docs PR.

Reply:
Option added to the PR, just waiting for core to get merged.

Reply:
PR has been merged.